cypress/integration/1227-customer_registry/366-application_administrator_adds_an_institution/add_institution.js

```diff
@@ -0,0 +1,35 @@
+import { Given, When, Then, And } from 'cypress-cucumber-preprocessor/steps';
+import { ADMIN_USER } from '../../../support/constants';
+import { INSTITUTION_FIELDS } from '../../../support/data_testid_constants';
+
+// Feature: Application Administrator adds an Institution
+
+//   @366
+//   Scenario: Application Administrator adds an Institution
+Given('that the user is logged in as Application Administrator', () => {

```

Comment:
syns det er litt synd å måtte duplisere dette, de 2 første stegene er jo samme som i #89, og det gjelder sikkert flere andre scenario også :/

Reply:
Vi burde vel å slått sammen de scenarioene

Reply:
at de ligger i samme feature tenker du?

Reply:
Ja, ser også at det er mulig å skrive common steps som ligger i en egen katalog, men er redd for at det da blir vanskelig å få oversikt